German NEO 2 and Colemack remap the caps key, so we need to remap the LWIN key
as well

diff --git a/symbols/de b/symbols/de
index b23aaf6..2087fe9 100644
--- a/symbols/de
+++ b/symbols/de
@@ -254,6 +254,7 @@ xkb_symbols "neo" {
     key.type[Group1]="THREE_LEVEL";
     key <BKSL> { [ ISO_Level3_Shift, ISO_Level3_Shift,  Caps_Lock ] };
     key <CAPS> { [ ISO_Level3_Shift, ISO_Level3_Shift,  Caps_Lock ] };
+    key <LWIN> { [ ISO_Level3_Shift, ISO_Level3_Shift,  Caps_Lock ] };
 
     // Definition of the so called Neo-Mod4
     key.type[Group1]="ONE_LEVEL";
diff --git a/symbols/us b/symbols/us
index c732de6..22f3190 100644
--- a/symbols/us
+++ b/symbols/us
@@ -723,6 +723,7 @@ xkb_symbols "colemak" {
     key <AB10> { [        slash,     question,    questiondown,       asciitilde ] };
 
     key <CAPS> { [    BackSpace,    BackSpace,       BackSpace,        BackSpace ] };
+    key <LWIN> { [    BackSpace,    BackSpace,       BackSpace,        BackSpace ] };
     key <LSGT> { [        minus,   underscore,          endash,           emdash ] };
     key <SPCE> { [        space,        space,           space,     nobreakspace ] };
 
